ideascube/mediacenter/forms.py

```diff
@@ -27,3 +30,30 @@ def save(self, commit=True):
         document.save()
         self.save_m2m()
         return document
+
+
+def CreateDocumentForm(path):
+    class NotFileDocumentForm(forms.ModelForm):
+
+        class Meta:
+            model = Document
+            exclude = ['original', 'preview']
+
+        original = forms.FilePathField(path=path)
+        preview = forms.FilePathField(path=path, required=False)
+
+        def save(self, commit=True):
```

Comment:
I don't think we want that.

I'd just use `**_` here, to catch all undesired kwargs and show we don't do anything with them.